back/spec/acceptance/initiatives_spec.rb

```diff
@@ -43,7 +43,7 @@
       expect(json_response[:data].size).to eq 0
     end
 
-    example 'List all initiatives which match one of the given topics' do
+    example 'List all initiatives which match one of the given topics', document: false do
```

Comment:
I have mixed feelings about removing this and the following examples from the docs. I agree that currently there are too many documented examples for the same endpoint. On the other hand, I think it can be valuable to document different use cases of making a request to the same endpoint.

Reply:
I agree. I actually found it hard to find many that I could clearly remove and just wanted to try a few in this one spec. In this case, based on the theory that with the listed params a dev could probably work out how to use the topics filter.

But yes, there are pros and cons in almost every case.